README.md

```diff
@@ -1,7 +1,7 @@
 # memmap
 
 A Rust library for cross-platform memory-mapped file IO. `memmap` requires Rust
-stable 1.1 or greater.
+stable 1.8 or greater.
```

Comment:
Could you also bump the version in .travis.yml?  I just turned on PR builds in travis, so it should be checked when you push a new revision.